src/app/screens/Game/components/Square/index.js

```diff
@@ -1,11 +1,19 @@
-import React, { Component } from 'react';
+import React from 'react';
+import PropTypes from 'prop-types';
 
-import style from './styles.css';
+import './styles.css';
 
-class Square extends Component {
-  render() {
-    return <button className={style.square}>{/* TODO */}</button>;
-  }
+function Square(props) {
+  return (
+    <button className="square" onClick={props.onClick}>
+      {props.value}
+    </button>
+  );
 }
 
+Square.propTypes = {
+  value: PropTypes.number,
```

Comment:
Value is a Required value because you are using it to render text

Reply:
Done